modules/serverless-event-delivery-component-behaviors.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// serverless/event_workflows/serverless-event-delivery.adoc
+
+[id="serverless-event-delivery-component-behaviors_{context}"]
+= Event delivery behavior for Knative Eventing components
+
+Different Knative Eventing components have their own behavior patterns that are followed for event delivery.
+
+== Delivery failure status codes
+
+The channel or broker receiver may respond with the following status codes if an event fails to be delivered:
+
+`500`:: This is a generic status code which means that the event was not delivered successfully.
+`404`:: This status code means that the channel or broker the event is being delivered to does not exist, or that the `Host` header is incorrect.
+`400`:: This status code means that the event being sent to the receiver is invalid.
+
+== Default channel and broker event delivery
+
+Developers can set event delivery parameters in the subscriber configuration to ensure that any events that fail to be delivered from `InMemoryChannel` channels or the `default` Knative broker dispatchers to an event sink are retried. You must also configure the dead letter sink for the subscriber if you want to provide a sink where events that are not eventually delivered can be stored, otherwise undelivered events are dropped.
```

Comment:
Is this only common to subscriptions to `InMemoryChannels` and brokers ? (what about subscriptionn on non-`InMemoryChannels`, don't they offer retries ?). Also,  I maybe adding a `respectively` would make sens to clear state that this are different scenarios. (I think this might even justify a bullet list here, to distinguish more clearly the scenarios and also maybe say that this is a common behaviour and the list might be only an example. In the list you then can also link back to the places where those options are described in detail for each command).

@openshift/team-knative-eventing are these options only available for "subscriptions" and not for "triggers" ?

Reply:
Also, shouldn't it be "subscription configuration" and not "subscriber configuration" ?

Reply:
We only support InMemoryChannels and Kafka channels at the minute, and we cover both of these, so I think covering additional non-InMemoryChannels that aren't Kafka is out of scope for now.
I don't think respectively applies here, since the options are the same for both components, we're not discussing two different types of configuration where it would make sense to say that.
These options aren't available for triggers yet afaik in our downstream release, I already asked @slinkydeveloper about this and we agreed just to cover subscriptions for now.

ACK re "subcriber", I'll change this to "subscription".